zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedReceivedSnapshot.java

```diff
@@ -194,17 +192,18 @@ private void checkSnapshotIdIsValid(final String snapshotId) throws SnapshotWrit
 
   private void writeReceivedSnapshotChunk(
       final SnapshotChunk snapshotChunk, final Path snapshotFile) throws SnapshotWriteException {
+
     try (final var channel =
-        FileChannel.open(snapshotFile, StandardOpenOption.CREATE_NEW, StandardOpenOption.WRITE)) {
+        FileChannel.open(snapshotFile, StandardOpenOption.WRITE, StandardOpenOption.CREATE)) {
       final ByteBuffer buffer = ByteBuffer.wrap(snapshotChunk.getContent());
 
       while (buffer.hasRemaining()) {
         final int newLimit = Math.min(buffer.capacity(), buffer.position() + BLOCK_SIZE);
+        channel.position(snapshotChunk.getFileBlockPosition() + buffer.position());
         channel.write(buffer.limit(newLimit));
         buffer.limit(buffer.capacity());
       }
 
-      channel.force(true);
```

Comment:
:question: What's the justification for removing this?

Reply:
did it for performance reasons but probably better to continue forcing to ensure the writes go well.